src/ext/java/org/opentripplanner/ext/transmodelapi/model/scalars/LinearFunctionFactory.java

```diff
@@ -9,49 +9,58 @@
 import graphql.schema.CoercingParseValueException;
 import graphql.schema.GraphQLScalarType;
 import java.util.Locale;
+import java.util.NoSuchElementException;
 import javax.annotation.Nonnull;
-import org.opentripplanner.routing.api.request.framework.CostLinearFunction;
+import org.opentripplanner.routing.api.request.framework.LinearFunctionSerialization;
 
-public class CostLinearFunctionFactory {
+public class LinearFunctionFactory {
 
   private static final String DOCUMENTATION =
-    "A linear function to calculate a value(y) based on a parameter (x): " +
-    "`y = f(x) = a + bx`. It allows setting both a constant(a) and a coefficient(b) and " +
-    "the use those in the computation. Format: `a + b x`. Example: `1800 + 2.0 x`";
+    """
+    A linear function`f(t)` to calculate a value based on a variable(t). The variable can be the
+    duration/time or cost for a leg or section of a path/itinerary. The function `f(t) = a + bt`
+    has a constant(a) and a coefficient(b) that will be used in OTP to compute `f(t)`.
+    
+    Format: `a + b t`. Example: `1800 + 2.0 t`. The constant `a` accept both whole numbers and
```

Comment:
Do you want to us use "1800" (seconds) as the example or the more readable "30m"?